services/stickers/contracts.go

```diff
@@ -0,0 +1,98 @@
+package stickers
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/rpc"
+	"github.com/status-im/status-go/services/ens/erc20"
+	"github.com/status-im/status-go/services/stickers/contracts"
+)
+
+var errorNotAvailableOnChainID = errors.New("not available for chainID")
+
+var stickerTypeByChainID = map[uint64]common.Address{
+	1: common.HexToAddress("0x0577215622f43a39f4bc9640806dfea9b10d2a36"), // mainnet
+	3: common.HexToAddress("0x8cc272396be7583c65bee82cd7b743c69a87287d"), // ropsten
+}
+
+var stickerMarketByChainID = map[uint64]common.Address{
+	1: common.HexToAddress("0x12824271339304d3a9f7e096e62a2a7e73b4a7e7"), // mainnet
+	3: common.HexToAddress("0x6CC7274aF9cE9572d22DFD8545Fb8c9C9Bcb48AD"), // ropsten
+}
+
+var stickerPackByChainID = map[uint64]common.Address{
+	1: common.HexToAddress("0x110101156e8F0743948B2A61aFcf3994A8Fb172e"), // mainnet
+	3: common.HexToAddress("0xf852198d0385c4b871e0b91804ecd47c6ba97351"), // ropsten
+}
+
+// TODO: refactor SNT. This is also used in ens service
```

Comment:
yes I think we need a shared place where we put all the contract addresses

Reply:
I was thinking of extracting all the smart contracts from both ens and stickerpacks into a common folder